pxd.c

```diff
@@ -1242,8 +2391,38 @@ int pxd_init(void)
 
 	printk(KERN_INFO "pxd: driver loaded version %s\n", gitversion);
 
+	printk(KERN_INFO"CPU %d/%d, NUMA nodes %d/%d\n", nr_cpu_ids, NR_CPUS, nr_node_ids, MAX_NUMNODES);
```

Comment:
numa support is available from very old 2.5 kernels. If numa support is not available on the architecture, then it will degenerate to a single numa node holding all memory. We shouldn't be optimizing for such systems anyways.